PyInstaller/hooks/hook-gi.py

```diff
@@ -0,0 +1 @@
+hiddenimports = ['gi._error']
```

Comment:
Please be more verbose in this top-level hook file and more comments about what is it `gi`, where/how to get it and what version and OS was used for testing this hook